lib/inspec/dependencies.rb

```diff
@@ -6,23 +6,25 @@
 require 'fileutils'
 require 'molinillo'
 require 'inspec/errors'
+require 'inspec/requirement'
 
 module Inspec
   class Resolver
     def self.resolve(requirements, vendor_index, cwd, opts = {})
       reqs = requirements.map do |req|
-        Requirement.from_metadata(req, cwd: cwd) ||
-          fail("Cannot initialize dependency: #{req}")
+        req = Inspec::Requirement.from_metadata(req, cwd: cwd)
+        req || fail("Cannot initialize dependency: #{req}")
       end
 
-      new(vendor_index, opts).resolve(reqs)
+      new(vendor_index, opts.merge(cwd: cwd)).resolve(reqs)
     end
 
     def initialize(vendor_index, opts = {})
       @logger = opts[:logger] || Logger.new(nil)
       @debug_mode = false # TODO: hardcoded for now, grab from options
 
       @vendor_index = vendor_index
+      @cwd = opts[:cwd] || './'
```

Comment:
Maybe we can reuse the profile path here to avoid adding a new option?  Overall I think we'll need to refactor a lot of this current working directory as it is currently very confusing.  Once we have the fetchers/vendoring more flushed out, I think this level of the code won't need to worry about this.